testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -3,7 +3,7 @@
 #
 # This feature depends on a JeOS image present on the proxy
 # Please make sure that the feature
-#     features/min_osimage_build_image.feature
```

Comment:
OK. We have in our documentation a list of the normalized prefixes.

Can you please add this new prefix there? It's in `guidelines.md`, section "Rules for features".